src/praisonai/praisonai/sandbox/modal.py

```
@@ -0,0 +1,374 @@
+                duration_seconds=duration,
+                started_at=started_at,
+                completed_at=completed_at,
+                metadata={
+                    "platform": "modal",
+                    "gpu": self.gpu,
+                    "language": language,
+                    "app_name": self.app_name,
+                }
+            )
+            
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)